mmdeploy/codebase/mmdet/deploy/object_detection_model.py

```diff
@@ -677,37 +677,51 @@ def __init__(self, backend: Backend, backend_files: Sequence[str],
         model_cfg = load_config(model_cfg)[0]
         self.model_cfg = model_cfg
 
-    def _get_bboxes(self, outputs):
+    def _get_bboxes(self, outputs, img_metas):
         from mmdet.models import build_head
         head_cfg = self.model_cfg._cfg_dict.model.bbox_head
         head = build_head(head_cfg)
         if head_cfg.type == 'YOLOXHead':
+            divisor = round(len(outputs) / 3)
             ret = head.get_bboxes(
-                outputs[:3],
-                outputs[3:6],
-                outputs[6:9], [dict(scale_factor=None)],
+                outputs[:divisor],
```

Comment:
Can we use `*outputs.chunk(3)`?

Reply:
`outputs` is a list of tensors, `chunk` might be not appropriate here.